proto/src/scim_v1/mod.rs

```diff
@@ -52,6 +53,46 @@ pub struct ScimEntryGetQuery {
     pub attributes: Option<Vec<Attribute>>,
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone, ToSchema)]
+pub enum ScimSchema {
+    #[serde(rename = "urn:ietf:params:scim:schemas:kanidm:sync:1:account")]
```

Comment:
should this be `urn:ietf:params:scim:schemas:kanidm:sync:1:Account` for case consistency?

Reply:
Possibly, but I think the lowercase is nicer. Given that today the "sync" paths are fully internal and version locked, we can change this safely.